release artifacts not meeting requirements

# Expected Behavior

Released artifacts are deployed to `ossrh`.

# Actual Behavior

Built artifacts are not meeting requirements.

# Steps to Reproduce

1. mvn release:prepare -Darguments="-Dmaven.test.skip=true"
2. mvn release:perform -Darguments="-Dmaven.test.skip=true

# Additional Information
```
[INFO] [ERROR] Rule failure while trying to close staging repository with ID "comhomeaway-1006".
[INFO] [ERROR]
[INFO] [ERROR] Nexus Staging Rules Failure Report
[INFO] [ERROR] ==================================
[INFO] [ERROR]
[INFO] [ERROR] Repository "comhomeaway-1006" failures
[INFO] [ERROR]   Rule "javadoc-staging" failures
[INFO] [ERROR]     * Missing: no javadoc jar found in folder '/com/homeaway/streamingplatform/stream-registry/0.3.1'
[INFO] [ERROR]   Rule "pom-staging" failures
[INFO] [ERROR]     * Invalid POM: /com/homeaway/streamingplatform/stream-registry-parent/0.3.1/stream-registry-parent-0.3.1.pom: Project description missing, Project URL missing, Developer information missing
[INFO] [ERROR]   Rule "sources-staging" failures
[INFO] [ERROR]     * Missing: no sources jar found in folder '/com/homeaway/streamingplatform/stream-registry/0.3.1'
[INFO] [ERROR]     * Missing: no sources jar found in folder '/com/homeaway/streamingplatform/schema-library-events/0.3.1'
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -4,10 +4,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-## [0.3.1] - 20181211
+## [0.3.1] - 20181212
 ## Changed
 - Specified `deploy` goal in maven-release-plugin
-- Fixed "site breaking release process" #47 
+- Fixed "site breaking release process" (#47)
+- Fixed ossrh release requirements  (#50)
 
 ## [0.3.0] - 20181211
 ### Added

--- a/assembly/pom.xml
+++ b/assembly/pom.xml
@@ -9,7 +9,7 @@
     <modelVersion>4.0.0</modelVersion>
 
     <artifactId>stream-registry</artifactId>
-    <name>Stream Registry Assembly</name>
+    <name>${project.groupId}:${project.artifactId}</name>
 
     <dependencyManagement>
         <dependencies>
@@ -48,6 +48,12 @@
 
     <build>
         <plugins>
+            <plugin>
+                <artifactId>maven-source-plugin</artifactId>
+            </plugin>
+            <plugin>
+                <artifactId>maven-javadoc-plugin</artifactId>
+            </plugin>
             <plugin>
                 <artifactId>maven-shade-plugin</artifactId>
             </plugin>

--- a/core/pom.xml
+++ b/core/pom.xml
@@ -9,7 +9,7 @@
     <modelVersion>4.0.0</modelVersion>
 
     <artifactId>stream-registry-core</artifactId>
-    <name>Stream Registry Core</name>
+    <name>${project.groupId}:${project.artifactId}</name>
     <packaging>jar</packaging>
 
     <dependencies>

--- a/kafka-infra-provider/pom.xml
+++ b/kafka-infra-provider/pom.xml
@@ -9,7 +9,7 @@
     <modelVersion>4.0.0</modelVersion>
 
     <artifactId>kafka-infra-provider</artifactId>
-    <name>Stream Registry Kafka Infra Provider</name>
+    <name>${project.groupId}:${project.artifactId}</name>
 
     <dependencies>
 

--- a/pom.xml
+++ b/pom.xml
@@ -28,6 +28,48 @@
             <organization>HomeAway</organization>
             <organizationUrl>https://www.homeaway.com</organizationUrl>
         </developer>
+        <developer>
+            <name>Nathan Walther</name>
+            <email>nwalther@homeaway.com</email>
+            <organization>HomeAway</organization>
+            <organizationUrl>https://www.homeaway.com</organizationUrl>
+        </developer>
+        <developer>
+            <name>Vinayak Ponangi</name>
+            <email>pponangi@homeaway.com</email>
+            <organization>HomeAway</organization>
+            <organizationUrl>https://www.homeaway.com</organizationUrl>
+        </developer>
+        <developer>
+            <name>Adam Westerman</name>
+            <email>awesterman@homeaway.com</email>
+            <organization>HomeAway</organization>
+            <organizationUrl>https://www.homeaway.com</organizationUrl>
+        </developer>
+        <developer>
+            <name>Jordan Moore</name>
+            <email>jomoore@homeaway.com</email>
+            <organization>HomeAway</organization>
+            <organizationUrl>https://www.homeaway.com</organizationUrl>
+        </developer>
+        <developer>
+            <name>Carlos Cordero</name>
+            <email>ccordero@homeaway.com</email>
+            <organization>HomeAway</organization>
+            <organizationUrl>https://www.homeaway.com</organizationUrl>
+        </developer>
+        <developer>
+            <name>Ishan Dikshit</name>
+            <email>idikshit@homeaway.com</email>
+            <organization>HomeAway</organization>
+            <organizationUrl>https://www.homeaway.com</organizationUrl>
+        </developer>
+        <developer>
+            <name>Rui Zhang</name>
+            <email>rzhang@homeaway.com</email>
+            <organization>HomeAway</organization>
+            <organizationUrl>https://www.homeaway.com</organizationUrl>
+        </developer>
     </developers>
 
     <packaging>pom</packaging>

--- a/pom.xml
+++ b/pom.xml
@@ -13,6 +13,7 @@
     </modules>
 
     <description>Stream Registry Parent POM</description>
+    <url>https://homeaway.github.io/stream-registry/</url>
 
     <developers>
         <developer>

--- a/pom.xml
+++ b/pom.xml
@@ -4,13 +4,30 @@
 
     <groupId>com.homeaway.streamingplatform</groupId>
     <artifactId>stream-registry-parent</artifactId>
-    <name>Stream Registry Parent POM</name>
+    <name>${project.groupId}:${project.artifactId}</name>
     <modules>
         <module>schema-library-events</module>
         <module>kafka-infra-provider</module>
         <module>core</module>
         <module>assembly</module>
     </modules>
+
+    <description>Stream Registry Parent POM</description>
+
+    <developers>
+        <developer>
+            <name>Arun Vasudevan</name>
+            <email>avasudevan@homeaway.com</email>
+            <organization>HomeAway</organization>
+            <organizationUrl>https://www.homeaway.com</organizationUrl>
+        </developer>
+        <developer>
+            <name>Prabhakaran Thatchinamoorthy</name>
+            <email>pthatchinamoorthy@homeaway.com</email>
+            <organization>HomeAway</organization>
+            <organizationUrl>https://www.homeaway.com</organizationUrl>
+        </developer>
+    </developers>
 
     <packaging>pom</packaging>
     <version>0.3.1-SNAPSHOT</version>
@@ -28,6 +45,7 @@
         <developerConnection>scm:git:ssh://git@github.com/HomeAway/stream-registry.git</developerConnection>
         <tag>HEAD</tag>
     </scm>
+
 
     <properties>
 
@@ -511,6 +529,8 @@
                         <releaseProfiles>signed</releaseProfiles>
                         <!-- to ensure we only deploy during release:perform -->
                         <goals>deploy</goals>
+                        <!-- to ensure that tags are of the right format -->
+                        <tagNameFormat>v@{project.version}</tagNameFormat>
                     </configuration>
                 </plugin>
                 <plugin>

--- a/schema-library-events/pom.xml
+++ b/schema-library-events/pom.xml
@@ -11,7 +11,7 @@
     <artifactId>schema-library-events</artifactId>
     <packaging>jar</packaging>
 
-    <name>Stream Registry Events</name>
+    <name>${project.groupId}:${project.artifactId}</name>
 
     <dependencies>
         <dependency>
@@ -52,6 +52,9 @@
                 <artifactId>build-helper-maven-plugin</artifactId>
             </plugin>
 
+            <plugin>
+                <artifactId>maven-source-plugin</artifactId>
+            </plugin>
             <!--To ignore avro code generated documentation warnings-->
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>

